src/commands/bootstrap/index.ts

```diff
@@ -0,0 +1,719 @@
+import yargs from 'yargs';
+import { CommonProps } from '../../types.js';
+import { writer } from '../../writer.js';
+import prompts, { Choice, InitialReturnValue } from 'prompts';
+import { validateNpmName } from './validate-pkg.js';
+import { basename, resolve } from 'path';
+import chalk from 'chalk';
+import { isCi } from '../../env.js';
+import { log } from '../../log.js';
+import { existsSync, writeFileSync } from 'fs';
+import { isFolderEmpty } from './is-folder-empty.js';
+import { EndpointType, ProjectListItem } from '@neondatabase/api-client';
+import { create } from '../projects.js';
```

Comment:
maybe replace with an api call to the create project endpoint. Would be strange if changes in "create" command would break this flow